api4/post.go

```diff
@@ -175,10 +175,16 @@ func getPostsForChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 	skipFetchThreads := r.URL.Query().Get("skipFetchThreads") == "true"
 	collapsedThreads := r.URL.Query().Get("collapsedThreads") == "true"
 	collapsedThreadsExtended := r.URL.Query().Get("collapsedThreadsExtended") == "true"
+	includeDeleted := r.URL.Query().Get("includeDeleted") == "true"
```

Comment:
Admittedly the existing Mattermost code isn't very good at being consistent with our query parameter casing, but everywhere else in the api code we use `include_deleted` instead of `includeDeleted`, so lets use `include_deleted`

Reply:
Done